core/server/worker/src/main/java/alluxio/web/UIFileBlockInfo.java

```diff
@@ -0,0 +1,118 @@
+/*
```

Comment:
we could put all the code that is currently duplicated code into a separate module

alternatively, we can (create a TODO to) remove the worker web UI logic that requires this code